src/WeekComponent/WeekComponent.css

```diff
@@ -41,66 +47,68 @@
   font-size: 18px;
   font-weight: bold;
  }
+ 
+ .switch-button {
+  width: 100%;
+  height: 32px;
+  align-items: center;
+  background-color: rgb(171, 180, 190);
+  border-radius: 12px;
+  cursor: pointer;
+  color: white;
+  font-family: sans-serif;
+  margin-bottom: 10px;
+  font-size: medium;
+ }
 
 .counting-seconds-data{
   display: flex;
   flex-direction: column;
-  padding: 3px 3px 3px 0px;
+  font-size: large;
+  width: 177px;
+  width: 195px;
 } 
 
 .exercise-data-container {
   display: flex;
   flex-direction: column;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  width: 100%;
 }
 
 .counting-seconds-container {
   display: flex;
   border: 1px solid rgb(238, 238, 238);
   background-color: rgb(238, 238, 238);
-  border-radius: 73px 0px 0px 0px;
-  width: 93%;
-  margin-left: 7%;
-  padding: 26px;
+  /* border-radius: 43px 0px 0px 0px; */
```

Comment:
Better to completely delete stuff instead of commenting. If we need it again we can get it from git.